src/mlpack/tests/ann/activation_functions_test.cpp

```diff
@@ -805,4 +805,22 @@ TEST_CASE("HyperSinhFunctionTest", "[ActivationFunctionsTest]")
 
   CheckActivationCorrect<HyperSinhFunction>(activationData, desiredActivation);
   CheckDerivativeCorrect<HyperSinhFunction>(desiredActivation, desiredDerivate);
+}
+/**
+ * Basic test of the Shifted-Softplus Function
+ */
+TEST_CASE("ShiftedSoftplusTest", "[ActivationFunctionsTest]")
```

Comment:
Can you add the same tests as for the other activation functions?

Reply:
@zoq what do you mean by same tests as for the other activation functions?

Reply:
Do i need to add the jacobian test as well??

Reply:
correct